phpstan.neon

```diff
@@ -256,3 +256,9 @@ parameters:
         - '#Method (.*?) returns bool type, so the name should start with is/has/was#'
 
         - '#Property Rector\\BetterPhpDocParser\\PhpDocNode\\Gedmo\\BlameableTagValueNode\:\:\$value has no typehint specified#'
+
+        # known value
+        -
+            message: "#^Parameter \\#1 \\$variable of class Rector\\\\Php70\\\\Dto\\\\VariableAssignPair constructor expects PhpParser\\\\Node\\\\Expr\\\\ArrayDimFetch\\|PhpParser\\\\Node\\\\Expr\\\\PropertyFetch\\|PhpParser\\\\Node\\\\Expr\\\\StaticPropertyFetch\\|PhpParser\\\\Node\\\\Expr\\\\Variable, PhpParser\\\\Node\\\\Expr given\\.$#"
+            count: 1
+            path: packages/Php70/src/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector.php
```

Comment:
No need for such detailed ignore, as it's hard to maintain in case of renaming file.
Message is much better, as in the line above